processor/formatters.go

```diff
@@ -275,35 +275,33 @@ func toHashDeep(input chan Result) string {
 	str.WriteString("%%%% HASHDEEP-1.0\n")
 	if !contains(Hash, "sha256") && !contains(Hash, "all") {
 		str.WriteString("%%%% size,md5,filename")
+	} else if contains(Hash, "sha256") && !contains(Hash, "all") && !contains(Hash, "md5") {
+		str.WriteString("%%%% size,sha256,filename")
 	} else {
 		str.WriteString("%%%% size,md5,sha256,filename")
 	}
 
-	if MTime {
```

Comment:
I might be wrong but I can't see where mtime is allowed in the hashdeep output format so I've dropped it here...